homeassistant/components/evohome/strings.json

```
@@ -6,7 +6,7 @@
       "fields": {
         "mode": {
           "name": "[%key:common::config_flow::data::mode%]",
-          "description": "Mode to set thermostat."
+          "description": "Mode to set the system."
```

```suggestion
          "description": "Mode to set the system to."
```
Sounds a bit more complete imo, what do you think?